package.json

```diff
@@ -29,5 +23,17 @@
   "jspm": {
     "main": "lib/VSS.SDK",
     "format": "global"
+  },
+  "types": "./typings/index.d.ts",
+  "dependencies": {
+    "@types/jquery": "^2.0.34",
+    "@types/jqueryui": "^1.11.31",
+    "@types/knockout": "^3.4.38",
+    "@types/q": "0.0.32",
+    "@types/react": "^0.14.54",
+    "@types/requirejs": "^2.1.28"
+  },
```

Comment:
I think we should add it (see https://github.com/facebook/react/blob/master/packages/react/package.json, for example)